R/tokens.R

```diff
@@ -449,6 +462,14 @@ tokens.tokens_xptr <-  function(x,
     if (!include_docvars)
         docvars(x) <- NULL
     
+    if (!identical(get_concatenator(x), concatenator)) {
+        #warning('concatenator changed from "', 
+        #        get_concatenator(x), '" to "', concatenator, '"', call. = FALSE)
+        set_types(x) <- stri_replace_all_fixed(get_types(x), get_concatenator(x), 
```

Comment:
Substitute the old concatenator in types with new one and updated the object meta.